app/views/admin/cms/transcripts/_form.html.erb

```diff
@@ -67,3 +77,53 @@
     <% end %>
   </div>
 </div>
+
+<script>
+$( function() {
+function split( val ) {
```

Comment:
best to try to namespace things, currently these are sitting at the top level, as global functions. you can namespace like:

```
var SpeakerAutocomplete = {
  split: function(val) {
    return val.split...
  },
  other_function: function() {
  }
};

// And to call...
SpeakerAutocomplete.split('value');
```